includes/Block/Web_Stories_Block.php

```diff
@@ -0,0 +1,409 @@
+<?php
+/**
+ * Class Web_Stories_Block.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Block;
+
+use Google\Web_Stories\Embed_Base;
+use Google\Web_Stories\Story_Query;
+use Google\Web_Stories\Tracking;
+use Google\Web_Stories\Story_Post_Type;
+use Google\Web_Stories\Traits\Assets;
+use WP_Post_Type;
+
+/**
+ * Latest Stories block class.
+ */
+class Web_Stories_Block {
+	use Assets;
+
+	/**
+	 * Script handle.
+	 *
+	 * @var string
+	 */
+	const SCRIPT_HANDLE = 'web-stories-block';
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	const BLOCK_NAME = 'web-stories/list';
+
+	/**
+	 * Current block's block attributes.
+	 *
+	 * @var array Block Attributes.
+	 */
+	protected $block_attributes = [];
+
+	/**
+	 * Maximum number of stories users can select
+	 *
+	 * @var int
+	 */
+	protected $max_num_of_stories = 20;
```

Comment:
Why not a `const`?

Reply:
☝️

Reply:
This sets the number of stories user can pick in `select stories` block type, I needed your input on whether to keep this fixed or should we add a filter that developers can use to modify the number? If we are adding a filter then in that case `const` won't make sense.

Reply:
I will leave this for @swissspidy to decide

Reply:
For now I've update it to `const` as we do not have the filter in place yet, if we need we can circle back and update.